src/include/rbd/librbd.hpp

```diff
@@ -337,6 +337,8 @@ class CEPH_RBD_API Image
   /* @param op_flags see librados.h constants beginning with LIBRADOS_OP_FLAG */
   int aio_write2(uint64_t off, size_t len, ceph::bufferlist& bl,
 		  RBD::AioCompletion *c, int op_flags);
+  int aio_write_traced(uint64_t off, size_t len, ceph::bufferlist& bl, RBD::AioCompletion *c,
+		  const struct blkin_trace_info *trace_info);
```

Comment:
Nit: The ```struct``` keyword isn't required -- and in C++ it would be better to pass the ```blkin_trace_info``` as ```const blkin_trace_info &``` instead of a pointer (it isn't optional and isn't an in/out param). Applies to ```aio_read_traced``` and ```aio_discard_traced````